packages/web/src/types.ts

```diff
@@ -158,9 +159,18 @@ interface SignWithLsig {
 	args?: Uint8Array[];
 }
 
+interface SignWithMultisig {
+	sign: SignType.MultiSignature;
+	mparams: WalletMultisigMetadata;
+	fromAccount?: AccountSDK;
```

Comment:
I think the `fromAccount` should be a list of `AccountSDK` since we will be signing the transaction with multiple accounts

Reply:
And I think it should be named `signers`

Reply:
No, we are using multi sig param for AccountSDK. One of the param of mparams contains the address array. So , we dont' need it here.